### ISSUE TITLE
[jss-plugin-expand] - borderBottom property does not expand correctly

### ISSUE THREAD

**Issue by bhupinderbola** [cssinjs/jss#1023]
__Expected behavior:__
```
borderBottom: {
  width: 12,
  color: "green"
}
```
should compile to

```
border-bottom-width: "12px";
border-bottom-color: "green";
```

__Describe the bug:__
```
borderBottom: {
  width: 12,
  color: "green"
} 
```

compiles to

```
border-bottom: "12px green";    
```

which sets border-bottom-style to 'initial'.

__Codesandbox link:__ 
https://codesandbox.io/s/ym5z831w8x

__Versions (please complete the following information):__
 - jss: 10.0.0-alpha.9
 - Browser [e.g. chrome, safari]: chrome
 - OS [e.g. Windows, macOS]: Windows

There is an issue with the documentation as well. As per https://cssinjs.org/jss-plugin-expand?v=v10.0.0-alpha.9

```
container: {
    border: {
      color: 'black',
      width: 1,
      style: 'solid'
    }
  }
```

should compile to

```
.container {
  border: 1px solid black;
}
```

but it compiles to 

```
.container {
  border-width: 1px;
  border-style: solid;
  border-color: black;
}
```

**Comment by JayaKrishnaNamburu** [comment]
Hey @HenriBeck , i would like to give a PR for this please 😄

**Comment by HenriBeck** [comment]
Yes, create a PR, and I will review it. If you have any questions about the code, leave a comment here.

**Comment by JayaKrishnaNamburu** [comment]
Hey @HenriBeck you can close this now

**Comment by HenriBeck** [comment]
Fixed in #1083
